utils/utils.go

```diff
@@ -315,6 +331,28 @@ func CustomizeLibertyEnv(pts *corev1.PodTemplateSpec, la *wlv1.WebSphereLibertyA
 		)
 	}
 
+	if IsFileBasedProbesEnabled(la) {
```

Comment:
This if-statement is slightly different from OLO:
https://github.com/OpenLiberty/open-liberty-operator/pull/788/files#diff-0df82f8689d3e8d881be0653ab4e9f8221c5835ec95a14930d11669e9c7f07efR331